python/ray/scripts/scripts.py

```diff
@@ -604,6 +605,21 @@ def start(
             '"CustomReseource2": 2}\''
         )
 
+    if plasma_store_socket_name is not None:
+        warnings.warn(
+            "plasma_store_socket_name is deprecated and will be removed. You are not "
+            "supposed to specify this parameter as it's internal.",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+    if raylet_socket_name is not None:
+        warnings.warn(
+            "raylet_socket_name is deprecated and will be removed. You are not "
+            "supposed to specify this parameter as it's internal.",
+            DeprecationWarning,
+            stacklevel=2,
```

Comment:
why is the stacklevel 2 not 3 here?

Reply:
With 2, it's able to surface to the public API already.